modules/installation-network-user-infra.adoc

```diff
@@ -271,6 +271,11 @@ ifdef::vsphere[]
 :!vsphere:
 endif::[]
 
+[discrete]
+== NTP configuration
+
+An {product-title} cluster that is installed in a restricted network is configured to use a public NTP server by default. To avoid clock skew, reconfigure the cluster to use a private NTP server instead. For more information, see the documentation for _Configuring chrony time service_.
```

Comment:
One tiny suggestion since this is the first reference to NTP in the module, maybe spell out the first one: 
Network Time Protocol (NTP)